dxr/query.py

```diff
@@ -209,6 +231,43 @@ def direct_result(self):
                 elif len(results) > 1:
                     return None
 
+    def _do_search_and_highlight(self, query, filters, is_line_query, offset, limit,
+                                 alphabetize=True):
+        """Perform a query using self.es_search and highlight the results,
+        returning a tuple (total count of results, list of results)
+
+        :arg query: valid ES query
+        :arg filters: nested list of filters
+        :arg is_line_query: whether to query lines (otherwise query files)
+        :arg offset: return results starting from this number
+        :arg limit: maximum number of results to return
+        :arg alphabetize: whether to alphabetize the results alphabetically by path and
```

Comment:
Everywhere else in the project, I've been naming boolean kwargs with a leading "should" or "is", like is_line_query above. Granted, it's a bit long here, but still.